packages/adminrouter/windows.build.ps1

```diff
@@ -24,5 +24,5 @@ Copy-Item -Recurse -Path "C:\Temp\openresty\openresty-1.15.8.2-win64\*" "$env:PK
 Copy-Item -Recurse -Path "C:\pkg\build\extra\src\*" "$env:PKG_PATH\etc\"
 Move-item -Path "$env:PKG_PATH\etc\includes" -Destination "$env:PKG_PATH\conf"
 Move-item -Path "$env:PKG_PATH\etc\mime.types" -Destination "$env:PKG_PATH\conf"
-Copy-Item "C:\pkg\build\dcos-adminrouter.nssm" "$env:PKG_PATH\conf\dcos-adminrouter.nssm.j2"
+Copy-Item "C:\pkg\build\adminrouter.nssm" "$env:PKG_PATH\conf\adminrouter.nssm.j2"
```

Comment:
This will conflict with the changes in https://github.com/dcos/dcos/pull/6641/files - I'd suggest we wait until it is merged and then rebase this change on top of an updated `master`

Reply:
ok, will waiting for merge

Reply:
I have done rebase